fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -180,6 +180,13 @@ def reset
         `xcrun simctl erase #{self.udid}`
         return
       end
+
+      def delete
+        UI.message("Deleting #{self}")
+        `xcrun simctl shutdown #{self.udid}` if self.state == "Booted"
```

Comment:
Is this necessary to check if the state is `Booted`? Can we just call shutdown regardless of state just in case something changes with states, we wont break of that string changes?

Reply:
I agree :)